src/main/groovy/com/getkeepsafe/dexcount/DexFile.groovy

```diff
@@ -80,16 +80,24 @@ class DexFile {
             }
         }
         // convert it to DEX format by using the Android dx tool
-        def androidSdkHome = System.getenv("ANDROID_HOME")
+        def androidSdkHome = DexMethodCountPlugin.sdkLocation
         if (androidSdkHome == null) {
             throw new Exception("ANDROID_HOME env variable not defined!")
         }
+
         def buildToolsSubDirs = new File(androidSdkHome, "build-tools")
         def dirs = buildToolsSubDirs.listFiles()
```

Comment:
It seems like `def dirs = buildToolsSubDirs.listFiles().sort { it.name }.reverse()` would be better here. Removes the need for the for loop below.